### ISSUE TITLE
Single-line comments lost

### ISSUE THREAD

**Issue by turadg** [txs1992/stylus-converter#65]
Here's a test case for the suite:
https://github.com/turadg/stylus-converter/tree/ta-comments-testcase

Relevant docs on [SCSS comments](http://sass-lang.com/documentation/file.SCSS_FOR_SASS_USERS.html#Comments) and [Stylus comments](http://stylus-lang.com/docs/comments.html).

The actual output loses all the single-line comments, regardless of the `--singlecomments` option.

**Comment by txs1992** [comment]
[issues 55](https://github.com/txs1992/stylus-converter/issues/55)

This issue was mentioned before, The stylus seems to have given up processing a single-line comment when converting ast. 

However, I did some special processing. Now I can support single-line comments and will be released in the next version.

**Comment by txs1992** [comment]
@turadg  This issue has been fixed in version 0.5.2.

**Comment by txs1992** [comment]
The problem is that the syntax for multiline comments is not supported in block mixins. Since stylus single-line comments are not available in ast, I convert them to multi-line comments with special tags, then convert them to scss before converting them to single-line comments. 

```
styl
// sign comments

-> scss
/* !#sign#! sign comments*/

-> scss
// sign comments
```
So here you want to keep single line comments, you need to put the comments outside the block mixins.

```
.Foo
  // single comment to end of line
  +some_mixin('m') 
    margin 0
```
